geoportailv3/templates/index.html

```diff
@@ -173,6 +166,7 @@ <h2 translate>infos</h2>
     <script src="${request.static_url('geoportailv3:static/js/main.js')}"></script>
     <script src="${request.static_url('%s/ngeo/utils/watchwatchers.js' % node_modules_path)}"></script>
     <script src="${request.static_url('%s/proj4/dist/proj4-src.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/ngeo/externs/typeahead.js' % node_modules_path)}"></script>
```

Comment:
Do not add a script tag to an externs file. You need to edit `package.json` and add a dependency to `typeahead.js` in there. See the ngeo PR.